### PR TITLE
added functions for mlebe

### PR DESCRIPTION
**Pull request by Jimmy2027**
Changes for compatibility with mlebe package:
- additional bias-correct node
- additional inputs for generic preprocessing function
- additional nodes for f/s-mask prediction

### CODE REVIEW

File: samri/pipelines/manipulations.py [hunk 1]
```diff
@@ -10,7 +10,8 @@ def collapse_nifti(in_dir, out_dir,
 	in_dir = os.path.abspath(os.path.expanduser(in_dir))
 	out_dir = os.path.abspath(os.path.expanduser(out_dir))
 	in_files = []
-	for root, dirs, files in os.walk(in_dir):
+	for root, dirs, files in os.walk(in_dir, followlinks=True):
+		for file in files:
```
Comment (TheChymera) [inline samri/pipelines/manipulations.py]: I don't understand, where does this for loop take place? There should be an indent after it, and some code using the variable `file`.

File: samri/pipelines/preprocess.py [hunk 2]
```diff
@@ -476,10 +484,33 @@ def generic(bids_base, template,
 				(s_warp, datasink, [('output_image', 'anat')]),
 				])
 
-		workflow_connections.extend([
-			(get_f_scan, get_s_scan, [('subject_session', 'selector')]),
-			(get_s_scan, s_warp, [('nii_name','output_image')]),
-			(get_s_scan, s_biascorrect, [('nii_path', 'input_image')]),
+		if model_prediction_mask == True:
```
Comment (TheChymera) [inline samri/pipelines/preprocess.py]: Why not just use `if model_prediction_mask:` ?

**Review by TheChymera — CHANGES_REQUESTED** [review]

File: samri/pipelines/manipulations.py (hunk 1 above)
Comment (Jimmy2027) [reply samri/pipelines/manipulations.py]: yes the `for file in files` loop must be a leftover, sorry. I meant to add the `followlinks=True` exclusively in order to be able to follow symbolic links.

```suggestion
```

File: samri/pipelines/preprocess.py (hunk 2 above)
Comment (Jimmy2027) [reply samri/pipelines/preprocess.py]: yes that would be cleaner

```suggestion
		if model_prediction_mask:
```

**Comment by TheChymera** [comment]
@Jimmy2027 could you please rebase and merge all commits into one?

**Pull request closed without merging**